analyzer/src/main/kotlin/managers/Pub.kt

```diff
@@ -246,28 +260,32 @@ class Pub(
 
             val parsePackagesResult = parseInstalledPackages(lockFile, labels, workingDir)
 
-            val gradleDefinitionFiles = gradleDefinitionFilesForPubDefinitionFiles.getValue(definitionFile).toList()
-
-            if (gradleFactory != null) {
-                if (gradleDefinitionFiles.isNotEmpty()) {
-                    val gradleDependencies = gradleDefinitionFiles.map {
-                        PackageManagerDependencyHandler.createPackageManagerDependency(
-                            packageManager = gradleFactory.type,
-                            definitionFile = VersionControlSystem.getPathInfo(it).path,
-                            scope = "releaseCompileClasspath",
-                            linkage = PackageLinkage.PROJECT_STATIC
+            if (!pubDependenciesOnly) {
+                val gradleDefinitionFiles = gradleDefinitionFilesForPubDefinitionFiles.getValue(definitionFile).toList()
+
+                if (!pubDependenciesOnly) {
```

Comment:
Is checking for `!pubDependenciesOnly` twice intended?

Reply:
Not really, something went wrong when rebasing.